charts/shoot-core/charts/calico/templates/calico.yaml

```diff
@@ -351,9 +358,27 @@ spec:
             - readiness
           periodSeconds: 10
 
+
+---
+
```

Comment:
Unused?

Reply:
Yes, we have not yet enabled calico-typha, but I wanted to keep the resource manifest up to date with the latest recommended one promoted by the projectcalico documentation itself.